SCClassLibrary/Common/Collections/windows/extString_windows.sc

```diff
@@ -3,8 +3,8 @@
 		format("start \"SuperCollider runInTerminal\" cmd.exe /k %", this.quote).unixCmd;
 	}
 
-	openOS {|aPath|
+	openOS {
 		// start "title" "command"
-		("start" + "\"SuperCollider\"" + this.quote).systemCmd
+		["start", "SuperCollider", this].unixCmd;
```

Comment:
I tested on Windows and the strings here need extra quotations (i.e. quotes need to be passed to the command). This works for me:
```supercollider
["start", "SuperCollider".quote, this.quote].unixCmd;
```

Reply:
Sorry for my lack of Windows knowledge, but just to make sure: have you tried it also with characters that need to be escaped? For example, does it work if the path includes a double quotes?
Otherwise could we go with `.shellQuote`?

Thanks for testing this @dyfer!!

Reply:
Just tried it - paths on Windows can't contain quotes, ([this](https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#:~:text=The%20following%20reserved,*%20(asterisk)) confirms it), so this example is noop regardless of openOS functionality.

Reply:
Awesome, thanks @dyfer! I put those `.quote` in there, and for me we are ready to merge :)